lib/bottles.rb

```diff
@@ -9,28 +12,23 @@ def verses(starting, ending)
   end
 
   def verse(number)
+    vessel = Vessel.new("bottle", "beer")
+    inventory = Inventory.new(vessel, number)
+
     case number
     when 0
-      "No more bottles of beer on the wall, " +
-      "no more bottles of beer.\n" +
-      "Go to the store and buy some more, " +
-      "99 bottles of beer on the wall.\n"
-    when 1
-      "1 bottle of beer on the wall, " +
-      "1 bottle of beer.\n" +
-      "Take it down and pass it around, " +
-      "no more bottles of beer on the wall.\n"
-    when 2
-      "2 bottles of beer on the wall, " +
-      "2 bottles of beer.\n" +
-      "Take one down and pass it around, " +
-      "1 bottle of beer on the wall.\n"
+      command = "Go to the store and buy some more"
+      next_inventory = Inventory.new(vessel, 99)
```

Comment:
When you initialize a new Inventory object and pass in an already-instantiated Vessel object, do all of these Inventory objects share the same Vessel, or are they copies? I.e., if I could set `vessel`'s `vessel_type` here to "cups", would all the Inventory's vessels be that one?